src/targets/awsLambdaLayer.ts

```diff
@@ -143,56 +143,145 @@ export class AwsLambdaLayerTarget extends BaseTarget {
       await this.artifactProvider.downloadArtifact(packageFiles[0])
     );
 
-    const publishRegion = async (currentRegion: string) => {
-      const lambda = new Lambda({ region: currentRegion });
-
-      const publishedLayer = await this.publishAwsLayer(lambda, {
-        Content: {
-          ZipFile: artifactBuffer,
-        },
-        LayerName: this.config.layerName,
-        CompatibleRuntimes: this.config.compatibleRuntimes,
-        LicenseInfo: this.config.license,
-      });
-
-      await this.addAwsLayerPermissions(lambda, {
-        LayerName: this.config.layerName,
-        VersionNumber: publishedLayer.Version,
-        StatementId: 'public',
-        Action: 'lambda:GetLayerVersion',
-        Principal: '*',
-      });
-
-      logger.info(`Published layer in ${currentRegion}:
-        ${publishedLayer.LayerVersionArn}`);
+    /** Creates symlinks to the new version file, and updates previous ones if needed. */
+    const createVersionSymlinks = (
```

Comment:
Could you explain why this (and another one below) is an anon function assigned to const and not a proper method?